benchexec/tools/sbt-fizzer.py

```diff
@@ -0,0 +1,148 @@
+import benchexec.tools.template
```

Comment:
We need a machine-readable license and copyright declaration, and would prefer if the license is Apache 2.0. Could you please add a header like in other BenchExec files?